code/src/components/CheckBox.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const CheckBox = (props) => {
+    const {nextQuestion, setPersonality, personality, setPerson, person } = props
+        return (
+            <section className='wrap'>
+                <div className='content-wrap checkbox'>
+                    <h1>Check yourself!</h1>
+                    <p>What personality applies to you?</p>
+                    <form className="form-checkbox">
+                        <label htmlFor='personality'>
+                            <input
+                                className='input-space'
+                                type='checkbox'
+                                checked={personality}
+                                onChange={setPersonality}
+                                id='personality'
+                                name='personality'
+                            />
+                            fearless
+                        </label>
+                        <label htmlFor='person'>
+                            <input
+                                className='input-space'
+                                type='checkbox'
+                                checked={person}
+                                onChange={setPerson}
+                                id='person'
+                                name='person'
+                            />
+                            judgemental
+                        </label>
```

Comment:
could probably be made a little bit more dry with a map and an array with the values since the structure is the same for the two checkbox elements